### ISSUE TITLE
JSON support

### ISSUE THREAD

**Issue by trlafleur** [pherting/streamdeck-mqtt#4]
Is there a way to display values from a JSON object??
Many MQTT settings are in JSON format...

thanks

**Comment by pherting** [comment]
Hello @trlafleur, what do you mean exactly?
At the moment the plugin is only to send commands in a mqtt que.
There is no option to subscribe some topics and grab informations.

It will help when you explain a little more what you need and what is your goal.

**Comment by trlafleur** [comment]
it would be very useful to have the ability to display (subscribe) to MQTT
data coming from sensors...

~~ _/) ~~~~ _/) ~~~~ _/) ~~~~ _/) ~~

Tom Lafleur

**Comment by pherting** [comment]
Yeah thats true, but at the moment there is nothing in planing for this feature.
But i like the idea and will code it maybee in future.

**Comment by Eucliwood090** [comment]
> it would be very useful to have the ability to display (subscribe) to MQTT data coming from sensors... ~~ _/) ~~~~ _/) 

Hello Thank you for your work, do you know if a version with the possibility of subscribe is planned?

**Comment by pherting** [comment]
> > it would be very useful to have the ability to display (subscribe) to MQTT data coming from sensors... ~~ _/) ~~~~ _/)
> 
> Hello Thank you for your work, do you know if a version with the possibility of subscribe is planned?

Hi, sorry for the late Answer!
At the moment there ist nothing planned to implement some features like this, sorry..
Maybe i found some free time next months =D
